### PR TITLE
Integrate Devphase

### PR DESCRIPTION
**Pull request by tolak**
This is PR is dedicated to integrating [Devphase](https://github.com/l00k/devphase) which is a dev tools build for Phat contract. Some attractive features that Devphase provided are:
- Run phala network (including node, pruntime, pherry) locally
- Phat contract deployment
- Phat contract unit test

The tasks:

- [x] Integrate Devphase
- [x] Write unit test for registry contract

- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,6 +20,47 @@ This repo contains several contracts, each of them can be compiled and deployed
...
+3. You can also dump the contract log from the log server driver with the same scripts:
+
+```sh
+ $ node src/dump-logs.js
+```
+
+4. Run tests
+
+```sh
+ $ yarn devphase test
```
Comment (h4x3rotab) [inline README.md:58]: Running test does not need to start the standalone stack

**Review by h4x3rotab — APPROVED** [review]
LGTM

File: README.md (hunk 1 above)
Comment (tolak) [reply README.md]: Fixed

File: tests/Registry.test.ts [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+                    "name": "Ethereum",
+                    "chainType": "Evm",
+                    "native": null,
+                    "stable": null,
+                    "endpoint": "https://rinkeby.infura.io/v3/6d61e7957c1c489ea8141e947447405b",
+                    "network": null,
+                })
+                .signAndSend(alice, {nonce: -1});
+            console.log('Register Ethereum', ethereumReg.toHuman());
+            await delay(1*1000);
```
Comment (h4x3rotab) [inline tests/Registry.test.ts:63]: Why delay?

**Pull request merged**